src/corecel/sys/Device.hh

```diff
@@ -118,10 +127,10 @@ class Device
     void create_streams(unsigned int num_streams) const;
 
     // Access a stream
-    Stream const& stream(StreamId) const;
+    Stream& stream(StreamId) const;
```

Comment:
I think it would be reasonable to for now make this non-const, or offer both versions:
```suggestion
    Stream& stream(StreamId);
```
since we virtually always access through `celeritas::device()`, which returns mutable, I think we can propagate the constness correctly.

Reply:
`celeritas::device()` returns a const ref, so we have to make this const as well.

Reply:
Aha. Sorry, guess I was thinking of the global `set_device`.